doc/api/crypto.md

```diff
@@ -1403,25 +1412,27 @@ input.on('readable', () => {
 });
 ```
 
-### crypto.createSign(algorithm)
+### crypto.createSign(algorithm[, options])
 <!-- YAML
 added: v0.1.92
 -->
 - `algorithm` {string}
+- `options` {object} combined [OpenSSL options][] and [`stream.transform` options][]
```

Comment:
Actually for Sign it's `stream.Writable`:

https://github.com/nodejs/node/blob/0d22858d67f5f8f7959a55ceca23adafe12827d5/lib/crypto.js#L285-L292